docs/content/en/tasks/monitor/_index.md

```diff
@@ -3,4 +3,203 @@ title: "Monitor K8ssandra"
 linkTitle: "Monitor"
 weight: 6
 description: "Access tools to monitor your Apache Cassandra® cluster running in Kubernetes."
----
\ No newline at end of file
+---
```

Comment:
It looks like these docs were removed in a previous PR, I'm not sure if that was a good idea since some folks still need to use MCAC given that older patch versions aren't supported by the new metrics agent. 

Just getting thoughts here, not insisting it be left it.